src/components/FAQ2/FAQ.scss

```diff
@@ -88,41 +88,44 @@
       cursor: pointer;
     }
   }
-  
+
   form {
     .input {
       position: relative;
-    
+
       i {
         color: $gray;
         position: absolute;
         left: 0px;
         top: 2px;
         font-size: 26px;
       }
-    
+
       input {
         font-size: 26px !important;
         margin-bottom: 26px !important;
         padding-left: 32px !important;
       }
-    
-      input::-webkit-input-placeholder { /* WebKit browsers */
+
+      input::-webkit-input-placeholder {
+        /* WebKit browsers */
         font-size: 26px !important;
```

Comment:
@IntegralTeam 
I know it is not your diff but if possible, please get rid of the 'important at some point'

Reply:
!important, itself, isn't bad. And when you're fixing user-agent styles it's not bad too.